README.md

```diff
@@ -5,6 +5,15 @@ https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?
 
 and print the github id, pr comment count and devstats contribution count as well.
 
+## Installation
+
+Run `make` to install Maintainers. Currently `make` will run the `hack/install-maintainers.sh` script.
+
+## Usage
+
+Maintainers looks for OWNERS files in the repository where you run `maintainers`. In the
```

Comment:
There are commands like `check-url` which don't need to interact with OWNERS files. Maybe we drop the Usage section or we could word it differently to be generic enough?

Reply:
I've made this more generic and not specific to OWNERS files. How does this new version look?